examples/bomex.jl

```diff
@@ -19,6 +19,7 @@ using Oceananigans.Units
 using AtmosphericProfilesLibrary
 using Printf
 using CairoMakie
+using DisplayAs #hide
```

Comment:
what does this do?

Reply:
> because of having to to run Literate in separate processes, for some reason I don't quite understand Literate also decided to start embedding all SVG images as raw HTML in the markdown files, making the resulting HTML files too large to be rendered by Documenter (the filesize limit could be bumped, but the fact that the images were included as blobs in the HTML isn't great on its own), so I used DisplayAs, as suggested in [Literate documentation](https://fredrikekre.github.io/Literate.jl/v2/tips/#notebook-filesize)